outlines/fsm/json_schema.py

```diff
@@ -119,18 +162,6 @@ def convert_json_schema_to_str(json_schema: Union[dict, str, Type[BaseModel]]) -
     return schema_str
 
 
-def _get_num_items_pattern(min_items, max_items, whitespace_pattern):
-    # Helper function for arrays and objects
-    min_items = int(min_items or 0)
-    if max_items is None:
-        return rf"{{{max(min_items - 1, 0)},}}"
-    else:
-        max_items = int(max_items)
-        if max_items < 1:
-            return None
-        return rf"{{{max(min_items - 1, 0)},{max_items - 1}}}"
-
-
 def validate_quantifiers(
```

Comment:
We should make this part of the class - also not have 40 lines of code for comparing two integers.